src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.GetRequiredWorkloads.CrossTargeting.targets

```
@@ -0,0 +1,33 @@
+<!--
+***********************************************************************************************
+Microsoft.NET.Sdk.GetRequiredWorkloads.CrossTargeting.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved.
+***********************************************************************************************
+-->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

```suggestion
<Project>
```

I think we don't need the xmlns as of MSBuild v15.0